packages/fxa-settings/src/index.tsx

```diff
@@ -30,12 +31,23 @@ try {
     <React.StrictMode>
       <AppContext.Provider value={appContext}>
         <AppErrorBoundary>
-          <App
-            {...{
-              flowQueryParams,
-              navigatorLanguages: navigator.languages,
-            }}
-          />
+          {
+            //hacky! needs a cleanup. sometimes we get a bool and sometimes a string.
+            flowQueryParams.showNewReactApp !== "false" ?
```

Comment:
Huh, do you know why this is the case? 🤔 Maybe you can `!!flowQueryParams.showNewReactApp` it.

Can we leave a comment here to detail what's happening (users in the experiment(s?) are showed this version vs the other just showing Settings)? Or maybe that'll be clear if `App` is renamed to `AppSettings`.